core/rawdb/accessors_chain.go

```diff
@@ -259,8 +259,7 @@ func WriteLastPivotNumber(db ethdb.KeyValueWriter, pivot uint64) {
 }
 
 // ReadTxIndexTail retrieves the number of oldest indexed block
-// whose transaction indices has been indexed. If the corresponding entry
-// is non-existent in database it means the indexing has been finished.
+// whose transaction indices has been indexed.
 func ReadTxIndexTail(db ethdb.KeyValueReader) *uint64 {
```

Comment:
The semantics here has changed, right? IIUC, the need for `nil` vs `zero` differentiation does no longer exist. If that's the case, we should just make it return a `uint64` instead of a pointer.

Reply:
Looked into it a bit. That change becomes quite substantial. Never mind